package.json

```diff
@@ -55,6 +54,7 @@
     "ws": "^7.2.3"
   },
   "devDependencies": {
+    "@types/koa": "^2.11.3",
```

Comment:
Putting `@types/koa` in normal deps is intentional because we export Koa types in the `Plugin` interface.

Reply:
Thanks, I got it.

The code has been changed back.